components/support/support-additional.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": 20000,
```

Comment:
> Where do the ids come from?

Hardcoded as I was trying to stay consistent with the opencollective data. And they're used for the `key` prop.

> Is there any chance these could clash in the future?

Potentially, yes. I tried to bump them pretty high to avoid that.

In general I like to avoid `key={ index }` everywhere, hence the ids. In our case, it currently doesn't matter too much for non-interactive components as a `render()` will never be called again on the client side. I'm ok with `key={ supporter.id || index }` for now and maybe shifting back to something like this, except using higher ids or uuids to prevent collisions, down the road.